Carta.py

```diff
@@ -320,67 +334,167 @@ def opponentRespondsInGrabPhase(self):
             else:
                 print("Carta.py: opponentRespondsInGrabPhase: opponent does not touch any card")
 
+    def removeCurrentGrabbingCard(self):
+        for card in self.grabbingCardsInPlay:
```

Comment:
you can keep track of "correct grabbing card" at an early stage, and then just remove the card. Then can skip this for-loop